### DISCUSSION TITLE
Download all EPG data for all channels

### DISCUSSION THREAD

**Discussion by ncat1** [iptv-org/community#1001] [Q&A] [answered]
I have a jellyfin server set up in my house with the iptv-org main playlist. I would like to set it up so that guide data is downloaded automatically for all supported channels, because I am trying to turn a $30 thin client PC into a ghetto set-top box. Is there a URL I can use that can download guide data for every supported channel? This is my first time asking a question on Github so I'm sorry if I broke some kind of etiquette rule.

**Comment by ncat1** [comment]
I should refine my question a bit. Is there a way to fetch guide data from every channel in the epg repo without fetching every single one manually?

**Reply by BellezaEmporium** [reply]
Creating a JS file that checks all configs for the EPG is doable. I had tried doing this once on a Jenkins backend.

**Comment by PopeyeTheSai10r** [comment]
You need to keep in mind that grabbing EPG for all channels will take some time and that some channels have multiple sources for the EPG Data.

https://iptv-org.github.io/?q=AEEast.us has 7 different sources for the Guides.

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/epg?tab=readme-ov-file#use-custom-channel-list
